### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by apprehensions**
+ different toml library
+ automatic dxvk installation
+ studio working
+ global programdirs
+ some shortening of functions

### CODE REVIEW

File: thirdparty.go [hunk 1]
```diff
@@ -19,9 +19,46 @@ const (
...
+func DxvkInstall(force bool) {
+	// Flatpak provides the graphics runtime, we cannot
+	// install it ourselves as Wine will crash.
+	//   org.winehq.Wine.DLLs.dxvk
+	// We are currently using org.winehq.Wine, which provides
+	// the aformentioned runtime included. We may use our own
+	// wine later.
+	if InFlatpak {
+		// Can we install dxvk? No, we cannot.
+		panic("DXVK must be managed by the flatpak.")
```
Comment (Thelolguy1) [inline thirdparty.go:53]: Just use log.Println and exit(1). panic() dumps some unnecessary debug info as well, and may not close file handles properly.

File: util.go [hunk 2]
```diff
@@ -24,7 +23,7 @@ func Errc(e error, message ...string) {
 		if message != nil {
 			log.Println(message)
 		}
-		log.Fatal(e)
+		panic(e)
```
Comment (Thelolguy1) [inline util.go:26]: panic() dumps unnecessary debug info and does not close file handlers properly. use log.Fatal, since that calls exit(1)

**Review by Thelolguy1 — CHANGES_REQUESTED** [review]

File: util.go (hunk 2 above)
Comment (apprehensions) [reply util.go:26]: Using `panic()` is intentional behavior, its how errors not handled by us should be handled, the debug information is required for further debugging when someone else comes to this crash.

Other than that, the panic screen looks nice.

Comment (apprehensions) [reply util.go:26]: > does not close file handlers properly. use log.Fatal, since that calls exit(1)

Doesn't the go runtime handle this ...?

Comment (Thelolguy1) [reply util.go:26]: OK makes sense. Errc should never(?) trigger anyway under normal conditions

File: thirdparty.go (hunk 1 above)
Comment (Thelolguy1) [reply thirdparty.go:53]: Put off until next wine update to flatpak (TBD)

**Review by Thelolguy1 — APPROVED** [review]

**Pull request merged**